core/container/container.go

```diff
@@ -78,6 +78,11 @@ type Router struct {
 	mutex           sync.Mutex
 }
 
+// Containers returns the map of containers that have been started.
+func (r *Router) Containers() map[string]Instance {
+	return r.containers
```

Comment:
1. Do we actually need a map returned?
2. Do we want to return a reference to the instance we're managing or a copy?

I think both could be addressed by returning a JIT allocated slice for the consumer.